GoogleForCreators/web-stories-wp
# Add labels for all inputs

## Summary
Adds missing aria labels for all inputs.
Adjust some existing labels to be more useful.


## Relevant Technical Choices

This PR removes the default values for labels since the "Standard input" aria label is not really useful, however, since it exists, a warning of missing label is not shown.

The default values are removed and all labels are changed to be required to enforce adding labels per use case, and not general labels.


## To-do



## User-facing changes
None.


## Testing Instructions

I
1. Add 1 of each element type (1 video, 1 image, 1 shape, 1 text)
2. Click on all the elements
3. Verify there are no missing label errors in the console.

II
1. Click on the Document tab
2. Verify there are no missing label errors in the console.



---



Fixes #742
